src/EditorFeatures/CSharpTest/RemoveUnnecessaryCast/RemoveUnnecessaryCastTests_AsTests2.cs

```diff
@@ -519,9 +519,9 @@ public static void Main()
 
         [WorkItem(545459, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/545459")]
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnnecessaryCast)]
-        public async Task RemoveUnneededCastInsideADelegateConstructor()
+        public async Task DoNotRemoveIllegalAsCastInsideADelegateConstructor()
         {
-            await TestInRegularAndScriptAsync(
```

Comment:
these cases were illegal code before.  we did remove the `as` case, but now do not.